developer-hub/src/polkadot_sdk/cumulus.rs

```diff
@@ -0,0 +1,130 @@
+//! # Cumulus
+//!
+//! Substrate provides a framework ([FRAME]) through which a blockchain node and runtime can easily
+//! be created. Cumulus aims to extend the same approach to creation of Polkadot parachains.
+//!
+//! > Cumulus clouds are shaped sort of like dots; together they form a system that is intricate,
+//! > beautiful and functional.
+//!
+//! ## Example: Runtime
+//!
+//! A Cumulus-based runtime is fairly similar to other [FRAME]-based runtimes. Most notably, the
+//! following changes are applied to a normal FRAME-based runtime to make it a Cumulus-based
+//! runtime:
+//!
+//! #### Cumulus Pallets
+//!
+//! A parachain runtime should use a number of pallets that are provided by Cumulus and Substrate.
+//! Notably:
+//!
+//! - [`frame-system`](frame::prelude::frame_system), like all FRAME-based runtimes.
+//! - [`cumulus_pallet_parachain_system`]
+//! - [`parachain_info`]
+#![doc = docify::embed!("./src/polkadot_sdk/cumulus.rs", system_pallets)]
+//!
+//! Given that all Cumulus-based runtimes use a simple Aura-based consensus mechanism, the following
+//! pallets also need to be added:
+//!
+//! - [`pallet_timestamp`]
+//! - [`pallet_aura`]
+//! - [`cumulus_pallet_aura_ext`]
+#![doc = docify::embed!("./src/polkadot_sdk/cumulus.rs", consensus_pallets)]
+//!
+//!
+//! Finally, a separate macro, similar to
+//! [`impl_runtime_api`](frame::runtime::prelude::impl_runtime_apis), which creates the default set
+//! of runtime APIs, will generate the parachain runtime's validation runtime API, also known as
+//! parachain validation function (PVF). Without this API, the relay chain is unable to validate
```

Comment:
nothing major, but would be cool to link to the glossary for certain terms eventually

Reply:
I have extended the glossary [here](https://github.com/paritytech/polkadot-sdk/pull/2273) (happy to get your review), once merged we can link

Reply:
I hope someday we can build some custom tooling to automate linking to glossary 💡